common/src/main/java/org/apache/uniffle/common/config/RssBaseConf.java

```diff
@@ -294,17 +294,29 @@ public class RssBaseConf extends RssConf {
                   + " first combining the username and the password with a colon (uniffle:uniffle123)"
                   + ", and then by encoding the resulting string in base64 (dW5pZmZsZTp1bmlmZmxlMTIz).");
 
-  public static final ConfigOption<String> RSS_STORAGE_WRITE_DATA_BUFFER_SIZE =
-      ConfigOptions.key("rss.storage.write.dataBufferSize")
+  public static final ConfigOption<String> RSS_STORAGE_LOCALFILE_WRITE_DATA_BUFFER_SIZE =
+      ConfigOptions.key("rss.storage.localfile.write.dataBufferSize")
           .stringType()
           .defaultValue("8k")
-          .withDescription("The buffer size to cache the write data content.");
+          .withDescription("The buffer size to cache the write data content for LOCALFILE.");
 
-  public static final ConfigOption<String> RSS_STORAGE_WRITE_INDEX_BUFFER_SIZE =
-      ConfigOptions.key("rss.storage.write.indexBufferSize")
+  public static final ConfigOption<String> RSS_STORAGE_LOCALFILE_WRITE_INDEX_BUFFER_SIZE =
+      ConfigOptions.key("rss.storage.localfile.write.indexBufferSize")
           .stringType()
           .defaultValue("8k")
-          .withDescription("The buffer size to cache the write index content.");
+          .withDescription("The buffer size to cache the write index content for LOCALFILE.");
+
+  public static final ConfigOption<String> RSS_STORAGE_HDFS_WRITE_DATA_BUFFER_SIZE =
+      ConfigOptions.key("rss.storage.hdfs.write.dataBufferSize")
+          .stringType()
+          .defaultValue("8k")
```

Comment:
Is `8k` ok for HDFS? Maybe we would better use Hadoop default config option before we have a benchmark.

Reply:
OK.